noxfile.py

```diff
@@ -64,24 +64,23 @@ def unit(session, repository, package, prerelease, protobuf_implementation, work
     # Pin mock due to https://github.com/googleapis/python-pubsub/issues/840
     session.install("mock==5.0.0", "pytest", "pytest-cov")
 
-    install_command = ["-e", f"{working_dir}/{downstream_dir}"]
     if prerelease:
+        session.install("-e", f"{working_dir}/{downstream_dir}")
```

Comment:
Why not move it outside the conditional logic like before and have a single `session.install` towards the end. It seems to be that we probably just want to remove `install_command.extend(["--no-deps"])`

Reply:
The `install_prerelease_dependencies` has several install commands where we install pre-release versions of dependencies. I wanted to make sure to install the dependencies of the downstream client before upgrading to the pre-release versions of dependencies. If we did this at the end , we may downgrade the pre-release dependencies